numpy/_core/src/umath/loops_arithmetic.dispatch.c.src

```diff
@@ -36,7 +36,7 @@
  *     q = TRUNC((n - (-dsign ) + (-nsign))/d) - (-qsign);
  ********************************************************************************/
 
-#if (defined(NPY_HAVE_VSX) && !defined(NPY_HAVE_VSX4)) || defined(NPY_HAVE_NEON)
+#if (defined(NPY_HAVE_VSX) && !defined(NPY_HAVE_VSX4)) || defined(NPY_HAVE_NEON) || defined(NPY_HAVE_LSX)
```

Comment:
This file does not seem to get built for LSX. You may have forgotten to add the build flags required to build this for LSX in https://github.com/numpy/numpy/blob/f3cca78729e301d4322c8f36fb44f3abb5d290d9/numpy/_core/meson.build#L887.

Reply:
Thanks for your suggestion, I have already made the modification.